frontend/src/components/ToolBar.vue

```diff
@@ -59,12 +59,12 @@ export default {
   background-color: purple;
 }
 
-@media only screen and (max-device-width: 768px) {
+@media only screen and (max-device-width: 1024px) {
   .tool-bar {
-    width: calc(100% - 51px);
-    min-width: calc(100% - 51px);
-    max-width: calc(100% - 51px);
-    height: 51px;
+    width: calc(100% - var(--header-height));
+    min-width: calc(100% - var(--header-height));
+    max-width: calc(100% - var(--header-height));
+    height: 52px;
     margin-left: 50px;
```

Comment:
I guess this should be `52px` (or `var(--header-height)`) to match the width/height of the Burger button.

Right now I see a little 2px gap on the right side of the toolbar.